src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md

```diff
@@ -3,18 +3,21 @@ title: SDK Interface
 sidebar_order: 12
 ---
 
-An interface which describes the Sentry SDK and its configuration used to
-capture and transmit the event.
+The SDK Interface describes the Sentry SDK and its configuration used to capture
+and transmit an event.
+
+## Attributes
 
 `name`:
 
-: **Required**. The name of the SDK. Its format is `sentry.ecosystem[.flavor]`
+: **Required**. The name of the SDK. The format is `sentry.ecosystem[.flavor]`
   where the _flavor_ is optional and should only be set if it has its own SDK.
 
 `version`:
 
-: **Required**. The semantic version of the SDK. The version should always be
-  sent without a `v` prefix.
+: **Required**. The version of the SDK. It should have the [Semantic
+  Versioning](https://semver.org) format `MAJOR.MINOR.PATCH`, without any
+  prefix.
```

Comment:
Maybe keep `v` as an example here, just to be explicit.

Reply:
Done.